code-examples/observer-demo/src/EconomistObserverDemo.java

```diff
@@ -0,0 +1,107 @@
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+public class EconomistObserverDemo {
+    public static void main(String[] args) {
+        Economist economist = new Economist();
+        Subscriber onlineSubscriber01 = new OnlineSubscription(economist);
```

Comment:
Intellij says that these Subscribers (mobileSubscriber01 ... up to ... physicalSubscriber05) are not being used. I wonder if this is an issue.